File: CS/注册界面.cs
// VBConversions Note: VB project level imports
using System.Collections.Generic;
using System;
using System.Linq;
using System.Drawing;
using System.Diagnostics;
using System.Threading.Tasks;
using System.Data;
using System.Xml.Linq;
using Microsoft.VisualBasic;
using System.Collections;
using System.Windows.Forms;
// End of VB project level imports

using System.Data.SqlClient;
using WindowsApp1;

namespace WindowsApp1
{
	public partial class 注册界面
	{
		public 注册界面()
		{
			InitializeComponent();
			
			//Added to support default instance behavour in C#
			if (defaultInstance == null)
				defaultInstance = this;
		}
		
#region Default Instance
		
		private static 注册界面 defaultInstance;
		
		/// <summary>
		/// Added by the VB.Net to C# Converter to support default instance behavour in C#
		/// </summary>
		public static 注册界面 Default
		{
			get
			{
				if (defaultInstance == null)
				{
					defaultInstance = new 注册界面();
					defaultInstance.FormClosed += new System.Windows.Forms.FormClosedEventHandler(defaultInstance_FormClosed);
				}
				
				return defaultInstance;
			}
			set
			{
				defaultInstance = value;
			}
		}
		
		static void defaultInstance_FormClosed(object sender, System.Windows.Forms.FormClosedEventArgs e)
		{
			defaultInstance = null;
		}
		
#endregion
		SqlConnection cn;
		SqlDataAdapter da;
		DataSet ds;
		SqlCommand cm;
		string sqlstr;
		public void Button2_Click(object sender, EventArgs e)
		{
			this.Close();
			登录界面.Default.Show();
		}
		
		public void Button1_Click(object sender, EventArgs e)
		{
			if (txtID.Text == "")
			{
				MessageBox.Show("请输入账号！");
			}
			if (txtPwd.Text == "")
			{
				MessageBox.Show("请输入密码！");
			}
			if (txtCpwd.Text == "")
			{
				MessageBox.Show("请再次输入密码！");
			}
			if (txtPwd.Text == "")
			{
				MessageBox.Show("请输入姓名！");
			}
			if (txtPwd.Text != txtCpwd.Text)
			{
				MessageBox.Show("两次输入密码不一致");
			}
			string cnStr = "Data Source=DESKTOP-DSP6URK\\SQLEXPRESS;Integrated Security=SSPI;Initial Catalog=changguan";
			cn = new SqlConnection(cnStr);
			cn.Open();
			if (radInside.Checked)
			{
				cm = new SqlCommand("select count(*) from Users where ID='" + txtID.Text + "'", cn);
				var num1 = Convert.ToInt32(cm.ExecuteScalar());
				if (num1 == 0)
				{

                    sqlstr = "insert into  Users values('" + txtID.Text + "','" + txtName.Text + "','" + "校内人员" + "; //,'" + txtPwd.Text + "')";
					cm = new SqlCommand(sqlstr, cn);
					cm.ExecuteNonQuery();
					cn.Close();
					MessageBox.Show("注册成功", "提示", MessageBoxButtons.OK, MessageBoxIcon.Information);
					登录界面.Default.Show();
					this.Visible = false;
				}
				else
				{
					MessageBox.Show("该账户已存在", "警告", MessageBoxButtons.OK, MessageBoxIcon.Information);
				}
			}
			else if (radOutside.Checked)
			{
				cm = new SqlCommand("select count(*) from Users where ID='" + txtID.Text + "'", cn);
				var num2 = Convert.ToInt32(cm.ExecuteScalar());
				if (num2 == 0)
				{
					
					sqlstr = "insert into  Users values('" + txtID.Text + "','" + txtName.Text + "','" + "校外人员" + "','" + txtPwd.Text + "')";
					cm = new SqlCommand(sqlstr, cn);
					cm.ExecuteNonQuery();
					cn.Close();
					MessageBox.Show("注册成功", "提示", MessageBoxButtons.OK, MessageBoxIcon.Information);
					登录界面.Default.Show();
					this.Visible = false;
				}
				else
				{
					MessageBox.Show("该账户已存在", "警告", MessageBoxButtons.OK, MessageBoxIcon.Information);
				}
				
			}
			else
			{
				MessageBox.Show("请选择身份！");
			}
			
		}
		
		public void 注册界面_Load(object sender, EventArgs e)
		{
			
		}
		
		public void RadioButton2_CheckedChanged(object sender, EventArgs e)
		{
			
			Label6.Text = "(手机号码注册)";
			
		}
		
		public void RadioButton1_CheckedChanged(object sender, EventArgs e)
		{
			Label6.Text = "(学号/工号注册)";
		}
	}
}

File: 体育场馆预约管理系统/WindowsApp1/注册界面.vb
Imports System.Data.SqlClient
Public Class 注册界面
    Dim cn As SqlConnection
    Dim da As SqlDataAdapter
    Dim ds As DataSet
    Dim cm As SqlCommand
    Dim sqlstr As String
    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles btnReturn.Click
        Me.Close()
        登录界面.Show()
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles btnRegister.Click
        If txtID.Text = "" Then
            MsgBox("请输入账号！")
        End If
        If txtPwd.Text = "" Then
            MsgBox("请输入密码！")
        End If
        If txtCpwd.Text = "" Then
            MsgBox("请再次输入密码！")
        End If
        If txtPwd.Text = "" Then
            MsgBox("请输入姓名！")
        End If
        If txtPwd.Text <> txtCpwd.Text Then
            MsgBox("两次输入密码不一致")
        End If
        Dim cnStr As String = "Data Source=DESKTOP-DSP6URK\SQLEXPRESS;Integrated Security=SSPI;Initial Catalog=changguan"
        cn = New SqlConnection(cnStr)
        cn.Open()
        If (radInside.Checked) Then
            cm = New SqlCommand("select count(*) from Users where ID='" + txtID.Text + "'", cn)
            Dim num1 = Convert.ToInt32(cm.ExecuteScalar())
            If (num1 = 0) Then

                sqlstr = "insert into  Users values('" + txtID.Text + "','" + txtName.Text + "','" + “校内人员" + "','" + txtPwd.Text + "')"
                cm = New SqlCommand(sqlstr, cn)
                cm.ExecuteNonQuery()
                cn.Close()
                MessageBox.Show("注册成功", "提示", MessageBoxButtons.OK, MessageBoxIcon.Information)
                登录界面.Show()
                Me.Visible = False
            Else
                MessageBox.Show("该账户已存在", "警告", MessageBoxButtons.OK, MessageBoxIcon.Information)
            End If
        ElseIf (radOutside.Checked) Then
            cm = New SqlCommand("select count(*) from Users where ID='" + txtID.Text + "'", cn)
            Dim num2 = Convert.ToInt32(cm.ExecuteScalar())
            If (num2 = 0) Then

                sqlstr = "insert into  Users values('" + txtID.Text + "','" + txtName.Text + "','" + "校外人员" + "','" + txtPwd.Text + "')"
                cm = New SqlCommand(sqlstr, cn)
                cm.ExecuteNonQuery()
                cn.Close()
                MessageBox.Show("注册成功", "提示", MessageBoxButtons.OK, MessageBoxIcon.Information)
                登录界面.Show()
                Me.Visible = False
            Else
                MessageBox.Show("该账户已存在", "警告", MessageBoxButtons.OK, MessageBoxIcon.Information)
            End If

        Else
            MsgBox("请选择身份！")
        End If

    End Sub

    Private Sub 注册界面_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub

    Private Sub RadioButton2_CheckedChanged(sender As Object, e As EventArgs) Handles radOutside.CheckedChanged

        Label6.Text = "(手机号码注册)"

    End Sub

    Private Sub RadioButton1_CheckedChanged(sender As Object, e As EventArgs) Handles radInside.CheckedChanged
        Label6.Text = "(学号/工号注册)"
    End Sub
End Class